src/Core/Repositories/LocalPackageRepository.cs

```diff
@@ -248,6 +248,9 @@ internal IEnumerable<IPackage> FindPackagesById(Func<string, IPackage> openPacka
         {
             foreach (var path in packagePaths)
             {
+                if(!FileNameMatchesPackageId(packageId, path))

```

Comment:
Follow the coding standards - always use curly braces for single line if (this repeats)